uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -366,32 +366,47 @@ Control {
                     }
                 }
             }
+            ToolSeparator {
+                Layout.fillWidth: true
+                Layout.columnSpan: 3
+                orientation: Qt.Horizontal
+            }
 
-            Button {
+            Label {
                 id: showAllFacilities
-                flat: true
-                text: "show all facilities"
+                text: '<a href=" "style="text-decoration: none">All sites</a>'
+                textFormat: Text.RichText
```

Comment:
using a `Label` with links to mimic the blue text shown on design

Reply:
No the color is not wrong, but from internal chat with Mark seems that the decided look is the blue text `All sites`. To get a blue color text without the overwriting of color in the tool, I thought of using the link functionality of a `Text` which in Calcite will output a blue text.
After click, no purple color.

Reply:
Gotcha!